app/controllers/candidate_interface/gcse/resolve_gcse_statement_comparability_path_concern.rb

```diff
@@ -0,0 +1,27 @@
+module CandidateInterface
+  module Gcse
+    module ResolveGcseStatementComparabilityPathConcern
+      def resolve_gcse_statement_comparability_path(subject)
+        case subject
+        when 'maths'
+          candidate_interface_new_gcse_maths_statement_comparability_path(subject:)
+        when 'science'
+          candidate_interface_new_gcse_science_statement_comparability_path(subject:)
+        when 'english'
+          candidate_interface_new_gcse_english_statement_comparability_path(subject:)
+        end
+      end
+
+      def resolve_gcse_edit_statement_comparability_path(subject)
+        case subject
```

Comment:
If we use `send` and interpolate the subject, we can remove the case statement, and also get a no route error if something unexpected comes in the `subject`

```ruby
send("candidate_interface_edit_gcse_#{subject}_statement_comparability_path")
```

Although the tradeoff is that the routes aren't as greppable

Reply:
In fact, that's what you're doing in app/views/candidate_interface/gcse/statement_comparability/edit.html.erb